test/fixtures/hello-cloud/redis.yml

```diff
@@ -63,6 +63,9 @@ apiVersion: apps/v1beta1
 kind: Deployment
 metadata:
   name: redis
+  labels:
```

Comment:
I don't mind these changes, but to confirm I'm not missing something: they weren't required, were they?